python_modules/dagster/dagster/components/resolved/model.py

```diff
@@ -102,6 +102,15 @@ def default(
             examples=examples,
         )
 
+    @staticmethod
+    def passthrough():
```

Comment:
kinda prefer `Resolver.noop()` but can't articulate why other than it's shorter and it feels more "standard"

Reply:
I don't think `noop` is clear. `passthrough` at least describes its purpose